examples/tests/invalid/lib/extern_above.w

```diff
@@ -0,0 +1,4 @@
+class Foo1 {
+  extern "../../valid/external_js.js" static getGreeting(name: str): str;
+       //^ must be a sub directory of the entrypoint
+}
```

Comment:
TODO for this PR:
This tests the case for directories, need to add a successful test case for an entrypoint (like the linked issue)